codectrl-gui/src/components/details_view_components/information_grid.rs

```diff
@@ -193,9 +198,19 @@ fn code_scroll(
 
             let mut layouter = |ui: &egui::Ui, string: &str, wrap_width: f32| {
                 // TODO: Hardcoded Langauge should be accepted from the log
-
+                let hash: u128 = xxhash(string.as_bytes());
                 let mut layout_job: egui::text::LayoutJob =
-                    code_highlighter(string, "rust"); // Change rust to log.language or something
+                    egui::text::LayoutJob::default();
+
+                if *code_hash == hash {
+                    layout_job = egui::text::LayoutJob::from(code_job.clone());
+                } else {
+                    let temp_job = code_highlighter(string, "rust"); // Change rust to log.language or something
```

Comment:
As you commented, please change to `log.language`

Reply:
log.language would break my code because this branch is behind on commits